core/tests/webdriverio_utils/BlogDashboardPage.js

```diff
@@ -148,6 +152,20 @@ var BlogDashboardPage = function() {
     await waitFor.invisibilityOfSuccessToast('Blog Post Saved Successfully.');
   };
 
+  this.updateAuthorDetails = async function(
+      authorName, authorBio
+  ) {
```

Comment:
```suggestion
  this.updateAuthorDetails = async function(authorName, authorBio) {
```

I think this might all fit on one line?

Reply:
Done